lib/api_browser/app/js/filters/tag_requirement.js

```diff
@@ -0,0 +1,13 @@
+app.filter('tagRequirement', function() {
+  return function(groups) {
+    if ( typeof groups === 'undefined' || groups.length == 0 ) {
+      return "";
+    }else if (groups.includes('all') ) {
+      var title = "required attribute";
+      return "<span title=\""+title+"\" style='font-weight:bold'><font color='red'>*</font></span>"
```

Comment:
This is better achieved with CSS classes. Ideally this would just be an empty `<span class="required" />` and event the star would be inserted via css.